lib/B/Deparse.t

```diff
@@ -3391,3 +3391,6 @@ $x = (return 'ok');
 $x //= (return 'ok');
 $x = exit 42;
 $x //= exit 42;
+####
+# preserve __LINE__ etc
+my $x = __LINE__;
```

Comment:
This could use tests for `__FILE__` and `__PACKAGE__` now that you're adding support for them.